config/_default/menus/menus.en.yaml

```diff
@@ -1,316 +1,322 @@
 main_left:
   - identifier: product
     name: Product
-    url: 'https://www.datadoghq.com/product/'
+    url: "https://www.datadoghq.com/product/"
     weight: -500
   - identifier: features
     name: Features
-    pre: '<img class=''menu-icon'' src=''https://imgix.datadoghq.com/img/navbar/menu/features.svg'' alt=''features icon''>'
-    url: 'https://www.datadoghq.com/product/'
+    pre: "<img class='menu-icon' src='https://imgix.datadoghq.com/img/navbar/menu/features.svg' alt='features icon'>"
+    url: "https://www.datadoghq.com/product/"
     weight: 1
     parent: product
   - identifier: integrations
     name: Integrations
-    pre: '<img class=''menu-icon'' src=''https://imgix.datadoghq.com/img/navbar/menu/integrations.svg'' alt=''integrations icon''>'
-    url: 'https://www.datadoghq.com/product/platform/integrations/'
+    pre: "<img class='menu-icon' src='https://imgix.datadoghq.com/img/navbar/menu/integrations.svg' alt='integrations icon'>"
+    url: "https://www.datadoghq.com/product/platform/integrations/"
     weight: 2
     parent: product
   - identifier: dashboarding
     name: Dashboards
-    pre: '<img class=''menu-icon'' src=''https://imgix.datadoghq.com/img/navbar/menu/dashboard.svg'' alt=''dashboard icon''>'
-    url: 'https://www.datadoghq.com/product/platform/dashboards/'
+    pre: "<img class='menu-icon' src='https://imgix.datadoghq.com/img/navbar/menu/dashboard.svg' alt='dashboard icon'>"
+    url: "https://www.datadoghq.com/product/platform/dashboards/"
     weight: 3
     parent: product
+  - identifier: infrastructure
+    name: Infrastructure Monitoring
+    pre: "<img class='menu-icon' src='https://imgix.datadoghq.com/img/navbar/menu/host-map-2.svg' alt='infra icon'>"
+    url: "product/infrastructure-monitoring/"
```

Comment:
hey @csantosdd can we update these infrastructure monitoring URLs to include the domain?   otherwise they will resolve to the docs site instead of corp.

Reply:
@bgdeutsch made the change! https://github.com/DataDog/documentation/pull/10750/commits/375e1743cc7f46bef853b9b02ffb850eb2f3cf2b